tokenizers/src/tokenizer/encoding.rs

```diff
@@ -22,6 +24,9 @@ pub struct Encoding {
     attention_mask: Vec<u32>,
     /// A list of overflowing Encoding generated when we got truncated
     overflowing: Vec<Encoding>,
+    /// Ranges of tokens covered by each sequence. If this is empty we consider
+    /// there is only one sequence in this Encoding, and that it covers the entire range.
```

Comment:
Is that a speed optimization ? We probably don't need that optimization until proven otherwise, no ?

Reply:
It just simplifies the code since we don't need to keep it updated when everything else changes.